awx/ui/client/lib/models/Credential.js

```diff
@@ -40,22 +45,73 @@ function assignInputGroupValues (inputs) {
     });
 }
 
+function setDependentResources (id) {
+    this.dependentResources = [
+        {
+            model: new ProjectModel(),
+            params: {
+                credential: id
+            }
+        },
+        {
+            model: new JobTemplateModel(),
+            params: {
+                credential: id,
+                ask_credential_on_launch: false
+            }
+        },
+        {
+            model: new InventoryModel(),
+            params: {
+                insights_credential: id
+            }
+        },
+        {
+            model: new InventorySourceModel(),
+            params: {
+                credential: id
+            }
+        }
+    ];
+}
+
 function CredentialModel (method, resource, graft) {
     BaseModel.call(this, 'credentials');
 
     this.Constructor = CredentialModel;
     this.createFormSchema = createFormSchema.bind(this);
     this.assignInputGroupValues = assignInputGroupValues.bind(this);
+    this.setDependentResources = setDependentResources.bind(this);
+    this.label = ModelsStrings.get('labels.CREDENTIAL');
 
     return this.create(method, resource, graft);
 }
 
-function CredentialModelLoader (_BaseModel_) {
+function CredentialModelLoader (
```

Comment:
I'd prefer to only drop down to a new line to avoid exceeding 100 chars per line.